psi4/psi4PluginCache.cmake.in

```diff
@@ -23,5 +23,7 @@ set(CMAKE_INSTALL_MESSAGE     "@CMAKE_INSTALL_MESSAGE@" CACHE STRING "")
 set(pybind11_DIR              "@pybind11_DIR@" CACHE PATH "")
 
 set(PYTHON_VERSION_MAJORMINOR "@Python_VERSION_MAJOR@.@Python_VERSION_MINOR@" CACHE STRING "")
+set(Python_VERSION_MAJORMINOR "@Python_VERSION_MAJOR@.@Python_VERSION_MINOR@" CACHE STRING "")
```

Comment:
Why are both cap versions needed?

Reply:
I'm not strictly sure both cap versions are needed for the python version. Both I suspect are needed for the python_executable so that old python detection (FindPythonLibsNew from pb11 that we ship with) and new python detection (native cmake when >3.15) are both adequately hinted. I didn't have to modify this file to get v2rdm to work both ways, but it seems a good precaution.